bin/sku.js

```diff
@@ -13,9 +15,38 @@ if (args.debug) {
   debug.enable(process.env.DEBUG);
 }
 
+let skipConfigure = false;
+let skipValidatePeerDeps = false;
+const packageJson = getPathFromCwd('./package.json');
+const packageJsonExists = fs.existsSync(packageJson);
+
+if (packageJsonExists) {
+  const {
+    skuSkipConfigure = false,
+    skuSkipValidatePeerDeps = false,
+  } = require(packageJson);
+  skipConfigure = skuSkipConfigure;
+  skipValidatePeerDeps = skuSkipValidatePeerDeps;
+}
+
 const configureProject = async () => {
-  const configure = require('../lib/configure');
-  await configure();
+  if (!skipConfigure) {
+    const configure = require('../lib/configure');
+    await configure();
+  } else {
+    log(`"skuSkipConfigure" set in ${packageJson}, skipping sku configuration`);
+  }
+};
+
+const validatePeerDeps = () => {
+  if (!skipValidatePeerDeps) {
+    // Intentionally not awaiting async function as it's just for logging warnings
+    _validatePeerDeps();
+  } else {
+    log(
+      `"skuSkipValidatePeerDeps" set in ${packageJson}, skipping sku peer dependency validation`,
+    );
+  }
```

Comment:
Same early exit suggestion:

```suggestion
  if (skipValidatePeerDeps) {
	  log(
	    `"skuSkipValidatePeerDeps" set in ${packageJson}, skipping sku peer dependency validation`,
  	);
  }
  // Intentionally not awaiting async function as it's just for logging warnings
  _validatePeerDeps();
```